docs/user/rstudio/ide/guide/ui/ui-panes.qmd

```diff
@@ -23,9 +23,9 @@ Additional source columns can be added or removed to the left of the 4 primary p
 
 The source pane allows users to view and edit various code-related files, such as .R, .rmd, .qmd, .py, .css, or general text files such as .txt or .md. By default it is the top-left panel and can be launched by opening any editable file in RStudio. Each additional file that is opened will be added as a new tab within the Source Pane.
 
-Additional source "columns" can be added from the **Global Options** > **Pane Layout** > **Add Source Column**, which provides the ability to work on two (or more) source files at a time within the Source Pane.
+Additional source "columns" can be added to the left of the four primary panes. To add additional source columns, from the RStudio menu: **Global Options** \> **Pane Layout** \> **Add Source Column**, which provides the ability to work on two (or more) source files at a time within the Source Pane.
```

Comment:
I am not sure why these are here - can you explain this?
```suggestion
Additional source "columns" can be added to the left of the four primary panes. To add additional source columns, from the RStudio menu: **Global Options** > **Pane Layout** > **Add Source Column**, which provides the ability to work on two (or more) source files at a time within the Source Pane.
```

Reply:
This is an effort to provide full context for screen reader folks who can't "see" the **add source column** button in the screenshot. I also want to specifically call out the ability to add source columns since it's a feature that MANY users are not aware of from my experience in training/webinars.

I think it's worth mentioning as I'm saying there are 4 primary panes, but adding source columns can add any number of additional "panes" to the view.

Reply:
I am totally fine with the verbiage and how you went about this. I just didn't understand why we had those back slashes there.

Reply:
Ah! Those are escape characters in Quarto, I believe it injects them to be safer at render time.

Reply:
Resolved this in latest commit.